sql/dbschema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE users (
+  userId INT PRIMARY KEY IDENTITY(1,1),
+  firstName VARCHAR(50) NOT NULL,
+  lastName VARCHAR(50) NOT NULL,
+  email VARCHAR(50) NOT NULL,
+  deleted INT DEFAULT 0 NOT NULL
+);
+
+CREATE TABLE tasks(
+  taskId INT PRIMARY KEY IDENTITY(1,1),
+  task VARCHAR(300),
+  createdTime DATETIME NOT NULL,
+  deleted INT NOT NULL DEFAULT 0
+);
+
+CREATE TABLE userTasks(
```

Comment:
Why do we need a separate table linking the users and the tasks? Is the relationship user:task 1:1, 1:* or *:*?

Reply:
I think it's a many-many relation. one user can have several tasks, and one task can be assigned to multiple users.
I can make it one-one if you think many-many is not necessary here.

Reply:
I would agree with Fahrrad, a user can have many tasks and the task can have multiple users. *:*
But we can also structure it that only a user can be assigned to different tasks 1:*

Reply:
After talking with the team we decided a 1:* relationship is appropriated compare to the *:* relationship

Reply:
I like having the two tables and not having to introduce the reference table, it simplifies queries for us and the teams who pick this up in following sprints.